src/payment/strategies/ppsdk/payment-processors/none-payment-processor.ts

```diff
@@ -10,11 +10,19 @@ export class NonePaymentProcessor implements PaymentProcessor {
         private _stepHandler: StepHandler
     ) {}
 
-    process({ paymentMethod, bigpayBaseUrl }: ProcessorSettings) {
+    process({ paymentMethod, bigpayBaseUrl, token }: ProcessorSettings) {
         const paymentMethodId = `${paymentMethod.id}.${paymentMethod.method}`;
         const body = { payment_method_id: paymentMethodId };
+        const options = {
+            credentials: false,
+            body,
+            headers: {
+                authorization: token,
+                'X-XSRF-TOKEN': null,
```

Comment:
I suggest introducing a new flag to request-sender library to turn off the default behaviour, e.g: `xsrf: false`. The reason is because with the current approach, the client needs to know that this specific header `X-XSRF-TOKEN` needs overriding. It'd be better if the option is declared explicitly as a part of the `RequestOption` interface instead.

Reply:
That would definitely be nicer, I was following the president here: https://github.com/bigcommerce/request-sender-js/pull/27 which established this `null`ing as a way to remove the `X-XSRF-TOKEN`.

Are we happy to move this improvement to a future ticket?